README.md

```diff
@@ -1,3 +1,28 @@
 # GoProfile
 
-GoProfile is an implementation to migrate profiles
\ No newline at end of file
+GoProfile is an implementation to migrate data between Users and Profiles identifying them by email.
+
+## Basic usage
+
+```sh
+  go run main.go --email-user ${email} --email-profile ${email} --token ${token}
+```
+
+## Flags
+
+| Flag            | Type   | Description                        |
+|-----------------|--------|------------------------------------|
+|`--token`        | string |Token to make petitions|
+|`--email-user`   | string |Email user from last database (CMS) |
+|`--email-profile` | string |Email profile from new dabatase (CMS)|
+|`--env`          | string |Environment to make petitions       |
```

Comment:
the environments name separated by pipes can be cleaner than the description:
`development|staging|production`